src/hardware/ipx.cpp

```diff
@@ -1045,10 +1045,18 @@ class IPXNET final : public Program {
 				WriteOut("Sending broadcast ping:\n\n");
 				pingSend();
 				ticks = GetTicks();
```

Comment:
Can move the `int64_t ticks;` and construct right at the point of the use: `int64_t ticks = GetTicks();`